packages/now-client/src/upload.ts

```diff
@@ -129,14 +146,23 @@ export default async function* upload(
         }
 
         if (err) {
-          return bail(err);
+          if (isClientNetworkError(err)) {
+
+            console.log('Network error, retrying: ' + err.message);
```

Comment:
These need to be `debug()` or at least `console.error()` otherwise it will mess up getting the deployment URL from stdout.